README.md

```diff
@@ -1 +1,26 @@
-# hw01
\ No newline at end of file
+# hw01
+## About Me
+HELLO! My name is **Jarret Angbazo** and I am a second-year MPP student at the *Harris School of Public Policy*. I was born in Nigeria and before moving to Chicago for graduate school, I grew up in Maryland. Chicago is the furthest west on the continental United States that I've lived.
+
+Quick two truths and a lie:
+
+1. I did this assignment in 5 minutes
+
+2. I don't hate the player, I hate the game
+
+3. I ran a personal best in the 5k this weekend
+
+Here is a [**link**](https://www.linkedin.com/in/jarretangbazo) to my LinkedIn page. There you will be able to find my employment and education histories.
+
+Thanks for stopping by.
+
+### Process
```

Comment:
Great reflection! In the future, put this in the comment field when you open the pull request rather than in the body of the assignment itself